server/utils/excelExport/poamExport.ts

```diff
@@ -374,7 +378,31 @@ export async function generatePoam(
   const recommendations = 20;
```

Comment:
This looks like you're trying to handroll an enum.  Enums under the hood are integer indexed I believe, so you could just make an enum for this and grab the values that way.

https://bobbyhadz.com/blog/typescript-convert-enum-to-string

Don't forget that enums are PascalCase for both their own name as well as the items defined in them.